src/db/index.ts

```
@@ -4,14 +4,12 @@ import postgres from "postgres";
 const setup = () => {
   if (!process.env.DATABASE_URL) {
     console.error("DATABASE_URL is not set");
-    return {
-      select: () => ({
-        from: () => [],
-      }),
-    };
+    throw new Error("DATABASE_URL is not set");
```

Wanted to throw error here so frontend could catch it and handle gracefully (rather than silently fail in backend)